src/app.js

```diff
@@ -1,7 +1,20 @@
 /* @flow */
 'use strict'
 
+// SIGINT and uncaughtException do not run cleanup
+// so we have an exitHandler to clean up the database connection
+function exitHandler (error) {
+  if (error) {
+    console.error('Exiting: ', error)
+    process.exit(1)
+  }
+  process.exit(0)
+}
+
 module.exports = require('./lib/app')
 if (!module.parent) {
+  process.on('SIGINT', exitHandler)
+  process.on('uncaughtException', exitHandler)
```

Comment:
Yeah, that's explaining why you don't want to use `process.on('uncaughtException'`. It's better to stick with the default behavior, which is just to exit, apparently without cleanup. The risk is that you end up suppressing the stacktrace and make it very hard to debug, which is what it looks like will happen with the current code.